### PR TITLE
Add doggie tokens

### PR DESCRIPTION
**Pull request by smk762**
ref 1: https://github.com/KomodoPlatform/coins/pull/438
ref 2: https://github.com/KomodoPlatform/coins/pull/439

### CODE REVIEW

**Comment by smk762** [comment]
ref: https://github.com/KomodoPlatform/coins/pull/440

**Review by tonymorony — APPROVED** [review]

**Comment by cipig** [comment]
the nomics_ids are all wrong... nomics_ids have the format `TKL` or `TKL2`, see eg https://nomics.com/assets/tkl2-tokel/widget, `data-base="TKL2"`, so nomics_id is TKL2 in this case

File: assets/config/0.5.4-coins.json [hunk 1]
```diff
@@ -690,6 +690,24 @@
     "active": false,
     "currently_enabled": false
   },
+  "BANANO-BEP20": {
+    "coin": "BANANO-BEP20",
+    "name": "Banano",
+    "nomics_id": "ban-banano",
```
Comment (smk762) [inline assets/config/0.5.4-coins.json:696]: https://nomics.com/assets/ban-banano

File: assets/config/0.5.4-coins.json [hunk 2]
```diff
@@ -997,6 +1015,24 @@
     "active": false,
     "currently_enabled": false
   },
+  "BONE-ERC20" : {
+    "coin": "BONE-ERC20",
+    "name": "Bone ShibaSwap",
+    "coinpaprika_id": "bone-bone-shibaswap",
+    "coingecko_id": "bone-shibaswap",
+    "nomics_id": "bone6-bone-shibaswap",
```
Comment (smk762) [inline assets/config/0.5.4-coins.json:1023]: https://nomics.com/assets/bone6-bone-shibaswap

File: assets/config/0.5.4-coins.json [hunk 3]
```diff
@@ -1731,6 +1767,24 @@
     "currently_enabled": false,
     "wallet_only": true
   },
+  "CUMMIES-BEP20": {
+    "coin": "CUMMIES-BEP20",
+    "name": "CumRocket",
+    "nomics_id": "cummies-cumrocket",
```
Comment (smk762) [inline assets/config/0.5.4-coins.json:1773]: https://nomics.com/assets/cummies-cumrocket

File: assets/config/0.5.4-coins.json [hunk 4]
```diff
@@ -2114,6 +2168,42 @@
     "active": false,
     "currently_enabled": false
   },
+  "DOGEDASH-BEP20": {
+    "coin": "DOGEDASH-BEP20",
+    "name": "Doge Dash",
+    "nomics_id": "dogedash-doge-dash",
```
Comment (smk762) [inline assets/config/0.5.4-coins.json:2174]: https://nomics.com/assets/dogedash-doge-dash

File: assets/config/0.5.4-coins.json (hunk 1 above)
Comment (cipig) [reply assets/config/0.5.4-coins.json:696]: https://nomics.com/assets/ban-banano/widget shows
`<div class="nomics-ticker-widget" data-name="Banano" data-base="BAN" data-quote="USD"></div><script src="https://widget.nomics.com/embed.js"></script>`
so `nomics_id` is `BAN`
on nomics, the ids are not in the URL

Comment (smk762) [reply assets/config/0.5.4-coins.json:696]: k, will review and repair in seperate PR

**Pull request merged**